yw_website/yw_db/tests.py

```diff
@@ -1,3 +1,15 @@
 from django.test import TestCase
+from django.test.client import Client
+from yw_db.models import Workflow
```

Comment:
Are we using a test database for this or do these tests get run using the actual database?